ansible/group_vars/all.yml

```diff
@@ -85,4 +85,4 @@ virtual_machines:
   garden-linux:
     project: sap-se-gcp-gardenlinux
     images:
-      - gardenlinux-gcp-gardener-prod-amd64-934-6-76ea662
+      - "{{ lookup('file', 'group_vars/gardenlinux-image.txt') }}"
```

Comment:
Do we ever wish to support more than one image (like we do for RHCOS)? It might be worth making `gardenlinux-image.txt` a json file (with just a list of versions in it) so we can add more images if necessary, but probably not worth doing in this PR

Reply:
I also considered using a yaml file and using the [include_vars](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/include_vars_module.html) module, but had some trouble testing it and decided to go the path of least resistance (just use a file and call it a day).

Both these approaches would be nice to haves once we automate the RHCOS update.

Reply:
As for the question on whether we want to support more than the latest version, no, I don't think we want to, latest one should be enough.